### PR TITLE
SALTO-1897/add go to service to jira

### PR DESCRIPTION
**Pull request by guyhamo**
supported types:
Configuration
Dashboard
FieldConfiguration
FieldConfigurationScheme
Filter
IssueTypeScheme
IssueTypeScreenScheme
PermissionScheme
ProjectCategory
Project
NotificationScheme
Resolution
Screen
Workflow
SecurityScheme
Status
WorkflowScheme
IssueType
AttachmentSettings
IssueLinkType
ProjectRole
TimeTrackingProvider
Automation
IssueEvent
Priority
ScreenScheme
Board
ProjectComponent
CustomFieldContext
Field
DashboardGadget
Webhook
SecurityLevel
---

_Additional context for reviewer_
im not 100% sure about the try catch implementation but i know i dont want this code to fail a fetch/deploy

---
_Release Notes_: 

---
_User Notifications_:

### CODE REVIEW

**Comment by alonstern** [comment]
I didn't review the code yet, there some additional URLs we can add

> ApplicationRole
/secure/project/EditProjectRole!default.jspa?id=<id>


>SecurityLevel
/secure/admin/EditIssueSecurityScheme!default.jspa?schemeId=<id>

>TimeTrackingProvider
/secure/admin/TimeTrackingAdmin.jspa
 
> Automation - not global automation
I think for all automation you can do
/jira/settings/automation#/rule/<id>

> IssueEvent - same url for all elements
I think we have a specific URL here
/secure/admin/EditEventType.jspa?eventTypeId=<id>

**Review by alonstern — COMMENTED** [review]
👌👌👌

File: packages/jira-adapter/src/filters/service_url/service_url_information.ts [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  const projectKey = instance.value.projects[0].projectId.resValue.value.key
+  return `/jira/software/projects/${projectKey}/settings/automate#/rule/${instance.value.id}`
+}
+
+const createBoardServiceUrl = (instance: InstanceElement): string =>
+  `/jira/software/projects/${instance.value.name.replace(' board', '')}/boards/${instance.value.id}`
+
+const createProjectComponentServiceUrl = (instance: InstanceElement): string => {
+  const parentKey = getParents(instance)[0].resValue.value.key
+  return `/plugins/servlet/project-config/${parentKey}/administer-components?filter=${instance.value.name}&orderDirection=DESC&orderField=NAME&page=1`
```
[the remark this answers is not in the document]
Comment (guyhamo) [reply packages/jira-adapter/src/filters/service_url/service_url_information.ts:44]: i cant reach parent with the service url filter.

File: packages/jira-adapter/src/filters/service_url/service_url_information.ts [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+const createFieldServiceUrl = (instance: InstanceElement): string | undefined => {
+  if (!instance.value.id.startsWith('customfield_')) {
+    return undefined
+  }
+  return `/secure/admin/EditCustomField!default.jspa?id=${instance.value.id.replace('customfield_', '')}`
+}
+
+const createDashboardGadgetServiceUrl = (instance: InstanceElement): string =>
+  `/jira/dashboards/${getParentId(instance)}?maximized=${instance.value.id}`
```
[the remark this answers is not in the document]
Comment (guyhamo) [reply packages/jira-adapter/src/filters/service_url/service_url_information.ts:64]: i cant reach parent with the service url filter.

**Review by alonstern — APPROVED** [review]
Make sure to rebase before you merge. I think you might have a "silent" conflict here

**Pull request merged**